cmd/thanos/compact.go

```diff
@@ -190,11 +195,17 @@ func runCompact(
 	}, func() float64 {
 		return consistencyDelay.Seconds()
 	})
+	deleteDelayMetric := prometheus.NewGaugeFunc(prometheus.GaugeOpts{
```

Comment:
Again is `Metric` suffix is really needed? IDE will show you what type it is.

Reply:
deleteDelay is already a variable, so what would be better in that case?